nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/PutAzureBlobStorage.java

```diff
@@ -73,11 +73,23 @@
             .required(true)
             .build();
 
+    public static final PropertyDescriptor CREATE_CONTAINER = new PropertyDescriptor.Builder()
+            .name("azure-create-container")
+            .displayName("Create Container")
+            .expressionLanguageSupported(ExpressionLanguageScope.NONE)
+            .required(true)
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .allowableValues("true", "false")
+            .defaultValue("true")
+            .description("Specifies whether to check if the container exists and to automatically create it if it does not.  Permission to list containers is required.  If false, this check is not made, but the Put operation will fail if the container does not exist.")
```

Comment:
You hit a checkstyle failure on this line. Can you split it into multiple strings at < 100?

Reply:
Running the build with `-Pcontrib-check` will make sure this executes or if you want a quick run through you can do `mvn checkstyle:checkstyle --quiet -Dcheckstyle.output.file=/dev/stdout -Dcheckstyle.output.format=plain`.

Reply:
Thanks for pointing me in the right direction.  I was having a problem with the -Pcontrib-check running originally, but it is working now.  Going to submit a change for this.